e2e/terraform/README.md

```diff
@@ -17,21 +17,21 @@ People who already have an AWS account can skip the above step.
 - Select *IAM* under *Security, Identity & Compliance*.
 - In the *Dashboard*, click *Users*.
 - Click *Add User* button.
-- In the *User name* textbox, enter *openebsuser01* as the username.
+- In the *User name* textbox, type the name of the user you want to create. For Example: *openebsuser*
```

Comment:
enter the name of the user you want to create. For example, *openebuser*

Reply:
@mahebbar, thanks. The new commit contains the changes as suggested.